doc/user/content/ops/monitoring.md

```diff
@@ -175,3 +175,76 @@ Configuration parameter | Value
 `prometheus_url`        | `http://<materialized host>/metrics`
 `namespace`             | Your choice
 `metrics`               | `[mz*]` to select all metrics, or a list of specific metrics
+
+## System Logs
+
+Materialize emits logs to the stream specified by the `--log-file` invocation
+parameter, documented at [cli](/cli).
+
+Materialize strives to make it easy to monitor logs, and so we have clear
+standards for each of our log levels. The high-level goal is that materialize
+running well should *never* emit `WARN` or `ERROR` level logs, every log at that
+level should be resolved.
```

Comment:
Bit of a pet peeve of mine on the terms, but can we call the noun "log message" or just "message"? 

"It logs logs" is grammatically correct but confusing to read. 

How's this (with an extra clarity fix on the health):
```suggestion
standards for each of our log levels. The high-level goal is that a healthy 
materialized should *never* emit `WARN` or `ERROR` level messages,
every message at that level should be resolved.
```

Reply:
could even get rid of the passive voice by saying "every message at that level needs attention" or something.

Reply:
Ah, I agree with this concern! I normally use the term `event` instead of message, and used that in other places and will do that here.

Reply:
I'd rather use "message"; I think it's much clearer!